### ISSUE TITLE
LibreElec.tv 9.2.x master

### ISSUE THREAD

**Issue by brunnels** [dtechsrv/LibreELEC.tv#1]
I've been working on getting S8X2 building on latest libreelec master.  Have you made any effort to do the same?

I'm currently getting 220 of 270 built before it stops at v4l-utils after adding some new patches to get things building with the gcc version they're using.

**Comment by dtechsrv** [comment]
Unfortunately all amlogic-specific packages and compiler flags have been removed from the LE-9.2 branch:
[https://github.com/LibreELEC/LibreELEC.tv/pull/3578/files](https://github.com/LibreELEC/LibreELEC.tv/pull/3578/files)

But the mainline linux kernel now have limited support for Meson8* SOCs (AMLMX).
I will try to compile a version from this: [https://github.com/chewitt/LibreELEC.tv/tree/amlogic](https://github.com/chewitt/LibreELEC.tv/tree/amlogic)
If it works, this source will be the base for my new attempt (LE-10.x).

**Comment by brunnels** [comment]
Yeah I saw the same.  I put things back but then started running into a lot of issues with the old kernel and the new gcc version.  Switching to mainline kernel would be ideal!

I can build or test too if you need.  I have an S812 M8-Plus.  Really it's this but same thing as far as I can tell. https://matricom.net/gbox-q2-android-tv/.  I also have a khadas vim1 which is S905X so can test to ensure things still work on S8X5 and S9X5.

**Comment by dtechsrv** [comment]
Now I have a fake M8S+ box with S805 SoC, that have 1GB RAM and 8 GB eMMC, and an old MXQ box with same hardware.
I'm trying to provide support for Meson8b (S805) with same kernel and LE9 repository packages.
The fake box works perfectly (with MXQ's dtb), but the MXQ cannot boot this kernel from the SD-card.
I will solder the RS232 TTL pins to find out what is the problem during the boot process.

**Comment by brunnels** [comment]
The problem I had with mine was it was android to start and wouldn't boot the sd card.  I needed to load an old android update package I found `LibreELEC-Amlogic.arm-M8SPlus-7.0.2-update.zip` by booting to recovery.  Then I was able to scp the new LibreELEC image I built from your repo to the /storage/.update folder.

**Comment by dtechsrv** [comment]
I wrote a short howto about the boot/installation/update steps into the Readme file. Did you tried the toothpick method?

**Comment by dtechsrv** [comment]
Okay, I found a solution for the current stable (9.2.6) compilation.
It was hard work, but it still works.

Please check my new repository: [https://github.com/dtechsrv/LibreELEC-AML](https://github.com/dtechsrv/LibreELEC-AML)

**Comment by brunnels** [comment]
Great work thank you. I’ll give it a go and let you know.
